tests/unit/snapshot-config.json

```diff
@@ -0,0 +1,17 @@
+{
+    "micro_file_name": "test_snapshot_computation",
+    "coupling_params": {
+        "parameter_file_name": "test_parameter.hdf5",
+        "read_data_names": {"macro-scalar-data": "scalar", "macro-vector-data": "vector"},
+        "write_data_names": {"micro-scalar-data": "scalar", "micro-vector-data": "vector"}
+    },
+    "simulation_params": {
+        "micro_dt": 1.0
+    },
+    "snapshot_params": {
+        "post_processing_file_name": "snapshot_post_processing"
+    },
+    "diagnostics": {
+        "output_micro_sim_solve_time": "True"
+    }
```

Comment:
Is this diagnostics output necessary for a test of the snapshot functionality?

Reply:
No